src/category_theory/shift.lean

```diff
@@ -133,14 +133,6 @@ notation f`⟦`n`⟧'`:80 := (shift_functor _ n).map f
 
 end defs
 
-section examples
-variables [has_shift C ℤ]
-
-example {X Y : C} (f : X ⟶ Y) : X⟦(1 : ℤ)⟧ ⟶ Y⟦1⟧ := f⟦1⟧'
-example {X Y : C} (f : X ⟶ Y) : X⟦(-2 : ℤ)⟧ ⟶ Y⟦-2⟧ := f⟦-2⟧'
-
-end examples
-
```

Comment:
Haha, with the reduction of imports this file can't even generate `add_monoid ℤ`. The example is not particularly valuable, it just demonstrates use of notation, and by now there are plenty of usages in mathlib.